### PR TITLE
code review

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: src/main/scala/nl/biopet/tools/sampletsvtoconfig/SamplesTsvToConfig.scala [hunk 1]
```diff
@@ -52,9 +52,10 @@ object SamplesTsvToConfig extends ToolCommand {
           if (libraryColumn != -1) Some(values(libraryColumn)) else None
 
         //FIXME: this is a workaround, should be removed after fixing #180
-        if (sample.head.isDigit || library.exists(_.head.isDigit))
-          throw new IllegalStateException(
-            "Sample or library may not start with a number")
+        //if (sample.head.isDigit || library.exists(_.head.isDigit))
+        //  throw new IllegalStateException(
+        //    "Sample or library may not start with a number")
+        // Fixed ?
```
Comment (ffinfo) [inline src/main/scala/nl/biopet/tools/sampletsvtoconfig/SamplesTsvToConfig.scala]: Not sure ;)

File: src/main/scala/nl/biopet/tools/sampletsvtoconfig/ArgsParser.scala [hunk 2]
```diff
@@ -8,7 +8,7 @@ class ArgsParser(cmdName: String) extends AbstractOptParser[Args](cmdName) {
   opt[File]('i', "inputFiles") unbounded () valueName "<file>" action {
     (x, c) =>
       c.copy(inputFiles = x :: c.inputFiles)
-  } text "Input must be a tsv file, first line is seen as header and must at least have a 'sample' column, 'library' column is optional, multiple files allowed"
+  } text "Input must be a tsv file, first line is seen as header and must at least have a 'sample' column, 'library' column is optional, multiple files allowed" // What is the syntax for adding multiple files. You have to specify the flag only once or multiple times?
```
Comment (ffinfo) [inline src/main/scala/nl/biopet/tools/sampletsvtoconfig/ArgsParser.scala]: Always multiple time, so `-i <file1> -i <file2>`

File: src/main/scala/nl/biopet/tools/sampletsvtoconfig/ArgsParser.scala [hunk 3]
```diff
@@ -18,5 +18,5 @@ class ArgsParser(cmdName: String) extends AbstractOptParser[Args](cmdName) {
   } text """
            |When the extension is .yml or .yaml the output is in yaml format, otherwise it is in json.
            |When no extension is given the output goes to stdout as yaml.
-         """.stripMargin
+         """.stripMargin // This is unclear. An outputfile is referenced but stdout is mentioned as well? So the output in stdout differs from that of the file?
```
Comment (ffinfo) [inline src/main/scala/nl/biopet/tools/sampletsvtoconfig/ArgsParser.scala]: should been when no **file** is given ;)

**Review by ffinfo — CHANGES_REQUESTED** [review]

File: src/main/scala/nl/biopet/tools/sampletsvtoconfig/SamplesTsvToConfig.scala (hunk 1 above)
Comment (rhpvorderman) [reply src/main/scala/nl/biopet/tools/sampletsvtoconfig/SamplesTsvToConfig.scala]: @ffinfo I have updated the branch to reflect the comments, but the commits do not show up here.

**Review by ffinfo — APPROVED** [review]

**Pull request merged**